openmetadata-spec/src/main/resources/json/schema/metadataIngestion/dbtconfig/dbtCloudConfig.json

```diff
@@ -26,6 +26,12 @@
       "description": "Optional configuration to update the description from DBT or not",
       "type": "boolean",
       "default": false
+    },
+    "dbtClassificationName": {
```

Comment:
Instead of passing the `dbtClassificationName` inside the security configuration of DBT, should it be part of `dbtPipeline.json` directly?

Reply:
hmmm I see we also have the `dbtUpdateDescriptions` inside each `dbt*config.json` 🤔

Reply:
makes sense will do that

Reply:
@pmbrull added the fields outside please re-review